### PR TITLE
Adding python-kaleido feedstock

### PR DESCRIPTION
**Pull request by jonmmease**
This package is the pure Python wrapper for the newly added `kaleido-core` package (https://github.com/conda-forge/staged-recipes/pull/12747). Splitting off the compiled core means that this package is simple and `noarch`.

Note: this feedstock pulls the pure Python package from the official Kaleido releases on GitHub (using official sha256). It doesn't pull from PyPI because the PyPI packages bundle the compiled chromium core that we've split off into the `kaleido-core` package on conda-forge.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] ~If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Thanks!

### CODE REVIEW

**Comment by jonmmease** [comment]
The linux build is failing because this package depends on `kaleido-core`, which depends on `sysroot_linux-64==2.17`, which requires the Centos 7 docker image to build.

I tried skipping the Linux CI build temporarily, but platform selectors aren't allowed in noarch packages, so I'm not sure if I can get this passing until the feedstock is created.  I can fix the build right away after the feedstock is created if you're willing to merge in its current state.

What do you think @conda-forge/help-python?  Thanks a lot!

File: recipes/python-kaleido/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  requires:
+    - plotly >=4.10,<5.0
+  imports:
+    - kaleido
+
+about:
+  home: https://github.com/plotly/Kaleido
+  license: MIT
+  license_family: MIT
+  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE.txt
```
Comment (dopplershift) [inline recipes/python-kaleido/meta.yaml:37]: Should be able to just do `LICENSE.txt`.

File: recipes/python-kaleido/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (dopplershift) [inline recipes/python-kaleido/LICENSE.txt:1]: Looks like this file is in the tarball, so you don't need to include it separately.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Couple minor things with the license. Since there's nothing wrong with the recipe for fixing the linux issues, I'm happy to merge and have them solved in the feedstock.

File: recipes/python-kaleido/meta.yaml (hunk 1 above)
Comment (jonmmease) [reply recipes/python-kaleido/meta.yaml]: :+1:

File: recipes/python-kaleido/LICENSE.txt (hunk 2 above)
Comment (jonmmease) [reply recipes/python-kaleido/LICENSE.txt:1]: Good point!

**Comment by jonmmease** [comment]
> Couple minor things with the license. Since there's nothing wrong with the recipe for fixing the linux issues, I'm happy to merge and have them solved in the feedstock.

Thanks @dopplershift, I made the requested updates

File: recipes/python-kaleido/LICENSE.txt (hunk 2 above)
Comment (dopplershift) [reply recipes/python-kaleido/LICENSE.txt:1]: Missed removing this file.

Comment (jonmmease) [reply recipes/python-kaleido/LICENSE.txt]: Right, thanks! 01f22f1

**Review by dopplershift — APPROVED** [review]

**Comment by jonmmease** [comment]
Thanks, @dopplershift!  Anything else needed?

**Comment by dopplershift** [comment]
Was just waiting on CI to finish.

**Pull request merged**